modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSet.macro

```diff
@@ -10,4 +63,18 @@ definition {
 			locator1 = "FrontendDataSet#ITEMS_VIEW_TYPE_DROPDOWN");
 	}
 
+	macro selectToFieldDates {
+		Click(
+			key_dateField = "To",
+			locator1 = "FrontendDataSetAdmin#DATEPICKER_BUTTON");
+
+		Click(
+			locator1 = "FrontendDataSetAdmin#CHOOSE_DATE_DATEPICKER_MENU",
+			value1 = ${key_dateNumber});
+
+		Click(
+			key_dateField = "To",
+			locator1 = "FrontendDataSetAdmin#DATEPICKER_BUTTON");
+	}
```

Comment:
Is it possible to use YUICalendarWidget.addDate() macro instead of creating a new one?

Reply:
I guess it's not possible :( because this calendar structure is different from the YUICalendarWidget structure. for example, this dataset calendar doesn't have the _yui3-calendar-content_ class or any other class that contains _yui3_ term.